src/app/shared/components/save-cluster-template/template.html

```diff
@@ -55,6 +55,12 @@
       </mat-radio-button>
     </mat-radio-group>
   </form>
+  <div *ngIf="showSSHKeyWarning"
+       class="km-dialog-warning"
+       fxLayoutAlign=" center">
+    <i class="km-icon-error"></i>
+    <div>SSH keys will not be saved in the cluster templates that are not in the project scope.</div>
```

Comment:
Not an expert here, but I think the `the` befor cluster templates could be deleted. 🤔 
`SSH keys will not be saved in cluster templates that are not in the project scope.`

Reply:
Other than that no other remarks from my side.

Reply:
You're right. Fixed.